dev-tools/release_notes.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Usage: release_notes.py <version> > RELEASE_NOTES.html
+
+Generates release notes for a Storm release by generating an HTML doc containing some introductory information about the
+ release with links to the Storm docs followed by a list of issues resolved in the release. The script will fail if it finds
+ any unresolved issues still marked with the target release. You should run this script after either resolving all issues or
+ moving outstanding issues to a later release.
+
+"""
+
+from jira import JIRA
```

Comment:
No, the script uses https://pypi.python.org/pypi/jira/. I'll add a comment to the top that users should install that module.